dataprep/eda/missing/compute/nullivariate.py

```diff
@@ -157,3 +230,59 @@ def missing_dendrogram(df: DataArray) -> Any:
     )
 
     return dendrogram
+
+
+def missing_col_cnt(df: DataArray) -> Any:
+    """Calculate how many columns contain missing values."""
+    nulls = df.nulls
+    rst = nulls.sum(0)
+    rst = rst[rst > 0].compute()
+    cnt = rst.shape[0]
+
+    return cnt
+
+
+def missing_row_cnt(df: DataArray) -> Any:
+    """Calculate how many rows contain missing values."""
+    nulls = df.nulls
+    rst = nulls.sum(1)
+    rst = rst[rst > 0].compute()
+    cnt = rst.shape[0]
+
+    return cnt
+
+
+def missing_most_col(df: DataArray) -> Tuple[int, float, List[Any]]:
+    """Find which column has the most number of missing values."""
+    nulls = df.nulls
+    col_sum = nulls.sum(axis=0).compute()
+    maximum = max(col_sum)
+    rate = maximum / df.shape[0]
+    cnt = sum(col_sum == maximum)
+    rst = np.where(col_sum == maximum)[0]
+    if rst.shape[0] > __most__:
+        rst = rst[:__most__]
+
+    return cnt, rate, rst
+
+
+def missing_most_row(df: DataArray) -> Tuple[int, float, List[Any]]:
```

Comment:
Could you add the meaning of input and output parameter in the docstring? E.g., is the input `df` a row or a dataframe; and the meaning of the three output values. The same comments also hold for `missing_most_col` function.